projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/linked-cases/components/linked-cases-table/linked-cases-to-table.component.ts

```diff
@@ -136,6 +137,30 @@ export class LinkedCasesToTableComponent implements OnInit, AfterViewInit {
     }
   }
 
+  public sortReasonCodes(searchCasesResponse): LinkedCasesResponse[] {
+    searchCasesResponse.forEach((item: any) => {
+      if(item && item.reasons && item.reasons.length){
+        item.reasons.forEach((reason) => {
+          reason.sortOrder = this.getReasonSortOrder(reason.value.Reason)
+        });
+        item.reasons = item.reasons.sort((a, b) => a.sortOrder - b.sortOrder);
+        item.sortOrder = item.reasons[0].sortOrder;
+      }
+    });
+    searchCasesResponse = searchCasesResponse && searchCasesResponse.sort((a, b) => a.sortOrder - b.sortOrder)
```

Comment:
@Kartheek-S - You can write this as `searchCasesResponse?.sort((a, b) => a.sortOrder - b.sortOrder)`